registration/registration.go

```diff
@@ -168,6 +177,49 @@ func (r *Registration) CertifierInfo() *CertifierConfig {
 	return r.cfg.Certifier
 }
 
+func (r *Registration) LoadTrustedPublicKeys() error {
+	if r.cfg.Certifier == nil {
+		return ErrCertificationIsNotSupported
+	}
+
+	r.trustedKeysMtx.Lock()
+	defer r.trustedKeysMtx.Unlock()
+
+	r.trustedPublicKeys = [][]byte{}
+
+	dirPath := r.cfg.Certifier.TrustedKeysDirPath
+	if dirPath == "" {
+		return ErrTrustedKeyDirPathIsNotSet
+	}
+
+	err := filepath.Walk(dirPath, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() || filepath.Ext(path) != ".key" {
+			return nil
+		}
+
+		keyData, err := os.ReadFile(path)
+		if err != nil {
+			return err
+		}
+
+		key := Base64Enc{}
+		if err = key.UnmarshalFlag(string(keyData)); err != nil {
+			return err
+		}
+
+		if len(key.Bytes()) != ed25519.PublicKeySize {
+			return ErrInvalidPublicKey
+		}
+
+		r.trustedPublicKeys = append(r.trustedPublicKeys, key.Bytes())
+		return nil
```

Comment:
NIT: How about adding an INFO log saying that the new keys were successfully loaded?